lib/builder.js

```diff
@@ -275,8 +275,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
     systemConfigFunc(cfg); 
   };
 
-  // jshint evil:true
-  eval(source.toString());
+  require('vm').runInNewContext(source.toString(), {System: System, console: console});
```

Comment:
I had a thought here. How about doing:

```javascript
var g = Object.create(global);
g.System = System;
```

That way we provide access to all globals, but deny writes.